templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  
+  <p>Are you sure you want to delete the manufacturer {{ manufacturer }}</p>
+  <p>All cars with this type manufacturer will be also delete</p>
```

Comment:
Grammatical Issue: Consider rephrasing to 'All cars with this manufacturer will also be deleted' for clarity and correctness.